main.py

```diff
@@ -1466,6 +1466,9 @@ def on_hover(self, point, hover_zone):
 
     def handle_response(self, response, point):
         debug(response)
+        if response is None:
+            # Flow returns None sometimes
```

Comment:
Do you mind creating an issue for the Flow language server and linking it in the comment?